lib/fluent/plugin/out_timestream.rb

```diff
@@ -6,23 +6,24 @@
 
 module Fluent
   module Plugin
-    # Raise when measure has empty value
-    class MeasureHasEmptyValueError < StandardError
-      def initialize(key_name = '')
-        super("measure has empty value. key name: #{key_name}")
+    # rubocop: disable Metrics/ClassLength
+    # Fluent plugin for Amazon Timestream
+    class TimestreamOutput < Fluent::Plugin::Output
+
+      # Raise when measure has empty value
+      class EmptyValueError < StandardError
+        def initialize(key_name = '')
+          super("measure has empty value. key name: #{key_name}")
```

Comment:
```suggestion
          super("Measure `#{key_name}` has empty value.")
```

個人デバグ用以外の例外・ログは大文字で始まる印象だけどどうだろう

Reply:
少なくとも fluent に関しては、他のログを見ると小文字開始が普通みたいです。
```
2021-03-09 11:34:49 +0900 [info]: #0 fluent/log.rb:329:info: starting fluentd worker pid=46179 ppid=46171 worker=0
2021-03-09 11:34:49 +0900 [debug]: #0 fluent/log.rb:308:debug: buffer started instance=70359106312600 stage_size=0 queue_size=0
2021-03-09 11:34:49 +0900 [debug]: #0 fluent/log.rb:308:debug: flush_thread actually running
2021-03-09 11:34:49 +0900 [debug]: #0 fluent/log.rb:308:debug: enqueue_thread actually running
2021-03-09 11:34:49 +0900 [trace]: #0 fluent/log.rb:286:trace: enqueueing all chunks in buffer instance=70359106312600
2021-03-09 11:34:49 +0900 [info]: #0 fluent/log.rb:329:info: following tail of /var/log/nginx/view.log
2021-03-09 11:34:49 +0900 [info]: #0 fluent/log.rb:329:info: fluentd worker is now running worker=0
```